awx/main/utils/named_url_graph.py

```diff
@@ -245,6 +244,8 @@ def _generate_configurations(nodes):
 
 
 def _dfs(configuration, model, graph, dead_ends, new_deadends, parents):
+    from django.contrib.contenttypes.models import ContentType
```

Comment:
`named_url_graph.py` is now imported and used in the app ready path. This is a bootstrapping phase and like any bootstrapping phase all the facilities aren't initialized. We have to be careful of what we import and when.
 
Delays this import.